addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -154,6 +154,9 @@ models.Order = models.Order.extend({
             line.set_dirty(false);
         });
         this.trigger('change',this);
+        if (this !== this.pos.get_order() && this.server_id) {
+            this.pos.sync_from_server(null, [this], this.uid);
```

Comment:
```suggestion
            this.pos.sync_from_server(null, [this], [this.uid]);
```
I think it's better if we do it like this, for less confusion.

Reply:
Hi @caburj. We found out that we needed to compare by reference after all. Even if the user opens the same order. The invalidated changes are still important since the opened order is technically not the same even if it has the same uid. In short, we want to prevent the additional sync only and only if the user does not close the table. 

Should I open a new PR? Thanks